docs/rest_api.md

```diff
@@ -1,8 +1,63 @@
+This file documents the HaaS REST API in detail.
+
+# How to read
+
+Each possible API call had a entry below containing:
```

Comment:
Quoting henn (2015-08-18 16:36:23)

>    s/had/has/

And further more `s/had a/has an/`. Fixed.